md_question_one.ipynb

```diff
@@ -0,0 +1,1005 @@
+{
```

Comment:
<p>question: why do we have review body words as components in both of the graphs? What does this tell us exactly?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepakkrish212/MATH327-Final/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Discussed. I will explain this in detail in the presentation and the importance of the variance.